webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
> On Jun 9, 2015, at 11:21 , Silvia Pfeiffer <notifications@github.com> wrote:
> 
> In webvtt.html:
> 
> > @@ -542,6 +542,17 @@
> >     </section>
> >  
> >     <section>
> > +    <h3>Unicode normalization</h3>
> > +
> > +    <p>This specification does not normalize Unicode text as part of its processing
> > +    requirements.</p>
> > +
> > +    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
> 
> David: As a UA dev, you are required not to normalize, because the spec does not normalize. Therefore, as a Web dev, don't expect the UA to normalize.

But if, as a web developer, I come across a UA that *does* normalize, even though it’s not required to, would I see a change of behavior (things that used to work, don’t)?

> 
> Would have some better wording to describe this?

I just want to make sure we’re not leaving an interop hole open.  Can one detect a difference between UAs that choose to normalize everything (though not required to) and those that don’t?  Is there a spec. and functional difference between ‘not required to normalize’ and ‘required not to normalize’?

David Singer
Manager, Software Standards, Apple Inc.